src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -7356,6 +7356,13 @@ protected internal void ImportModuleMembers(PSModuleInfo sourceModule, string pr
                 message = StringUtil.Format(Modules.ImportingFunction, prefixedName);
                 cmdlet.WriteVerbose(message);
             }
+            else
+            {
+                string manifestFileName = Path.GetFileName(cmdlet.Context.PreviousModuleProcessed);
```

Comment:
NIT: `filename` is one word, so you shouldn't capitalize the `N`

Reply:
Done. :)